src/lib/one-graph/cli-client.js

```diff
@@ -718,29 +820,80 @@ const generateSessionName = () => {
   return sessionName
 }
 
+/**
+ * Mark a session as inactive so it doesn't show up in any UI lists, and potentially becomes available to GC later
```

Comment:
Is this doctoring correct? And the name would probably be better as `idempotentlyUpdateSessionSchemaIdFromLockfile`